CHANGELOG.md

```diff
@@ -68,6 +68,9 @@ First bugfix release for version 2.1.0.
 - Renamed "All Data Offers" to "Data Catalogue" ([#52](https://github.com/sovity/authority-portal/issues/52))
 - Adjusted the ordering of users in the organization details page ([#98](https://github.com/sovity/authority-portal/issues/98))
 - Adjusted the ordering of organizations in the organization overview page ([#49](https://github.com/sovity/authority-portal/issues/49))
+- Restrict user re-/deactivation to respective statuses ([#82](https://github.com/sovity/authority-portal/issues/82))
+- Fixed MDS logo, footer (in compact view) ([#57](https://github.com/sovity/authority-portal/issues/57))
+- Fixed text overlapping in fullscreen pages ([#80](https://github.com/sovity/authority-portal/issues/80))
```

Comment:
Wrong section of the changelog. You need to put those under _unreleased_ instead of _2.1.0_ which is already released